pkg/collector/corechecks/networkpath/config.go

```diff
@@ -16,11 +16,15 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-const defaultCheckInterval time.Duration = 1 * time.Minute
+const (
+	defaultCheckInterval time.Duration = 1 * time.Minute
+	defaultTimeout       time.Duration = 10 * time.Second
+)
 
 // InitConfig is used to deserialize integration init config
 type InitConfig struct {
-	MinCollectionInterval int `yaml:"min_collection_interval"`
+	MinCollectionInterval int64 `yaml:"min_collection_interval"`
+	TimeoutMs             int64 `yaml:"timeout"` // millisecond
```

Comment:
```suggestion
	TimeoutMs             int64 `yaml:"timeout"`
```

no need to millisecond comment since the variable is already suffixed with Ms